src/sqlancer/common/oracle/DQEBase.java

```diff
@@ -0,0 +1,107 @@
+package sqlancer.common.oracle;
+
+import java.sql.SQLException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import sqlancer.Main;
+import sqlancer.MainOptions;
+import sqlancer.SQLConnection;
+import sqlancer.SQLGlobalState;
+import sqlancer.common.query.ExpectedErrors;
+import sqlancer.common.query.SQLQueryAdapter;
+import sqlancer.common.query.SQLQueryError;
+import sqlancer.common.schema.AbstractRelationalTable;
+
+public abstract class DQEBase<S extends SQLGlobalState<?, ?>> {
+
+    public static String COLUMN_ROWID = "rowId";
```

Comment:
for those not familiar with DQE, it would be great to explain important bits in the common base classes